_posts/2024-10-18-[paper]-SafeAr

```diff
@@ -0,0 +1,6 @@
+---
+layout: post
+author: Brendan David-John
+---
+
+On October 18, 2024 "Understanding the long-term impact and perceptions of privacy-enhancing technologies for bystander obscuration in AR" was accepted as a SafeAR workshop paper at ISMAR 2024. The project is a collaboration with Dr. Bo Ji (VT) and Dr. Evan Selinger (RIT).

```

Comment:
I'm uncertain about the data and classification as a workshop paper for this one?

Reply:
I would remove the 'On October 18, 2024'. I would then add "with support from CCI and the VT Center for HCI." to the last sentence.